doc/APIGuide.md

```diff
@@ -57,28 +57,28 @@ model.getMassCenter(state); // Need to know pose of model.
 
 ## Object {#object}
 
-[Object](@ref OpenSim::Object) is the base class for all OpenSim Components, general containers like Set, and Tools that require serialization - that is having their attributes written out to file. Objects user Properties to serialize (write to file) and deserialize (read from file) their attributes, which are parameters and settings that define the behavior of the Object.
+[Object](@ref OpenSim::Object) is the base class for all OpenSim Components, general containers like Set, and Tools that require serialization - that is having their attributes written out to file. Objects use Properties to serialize (write to file) and deserialize (read from file) their attributes, which are parameters and settings that define the behavior of the Object.
 
 ## Component {#component}
 
 A [Component](@ref OpenSim::Component) is the basic “unit” of modelling and computing in OpenSim. It is our abstraction to capture a fundamental unit of computing and/or modeling used to compose a Model and form computational system of equations. It is the base class that provides the supporting infrastructure for building up a hierarchically structured model with interconnections and numerous inputs and outputs, but is represented by a single computationally efficient (mathematical/numerical) system. That computational system is a SimTK::MultibodySystem and it is the computer implementation (think system of equations) of the multibody kinematics and dynamics plus any additional dynamics (ordinary differential equations, e.g. muscle activation dynamics) that need to be solved simultaneously. Any conceptual element that has to compute something typically requires the System and must be a Component in OpenSim. Component also provides the resources for reading and writing (also known as serializing, which it inherits from Object) the attributes/parameters that define the behavior of the component and adding in its dynamics (equations). The serializable attributes of a Component are called properties in OpenSim. A Component has the following user-facing attributes:
 
-1. Properties: Constant parameters (e.g., a Body’s mass).
+1. Properties: Built-in parameters (e.g., a Body’s mass).
```

Comment:
Why do you prefer `built-in` here? I like `constant` because it makes a distinction with state variables which are time-varying, and because I'm not sure what `built-in` means.

Reply:
I guess constant is in the eyes of the beholder. Constant gives the connotation to me that they can't be changed, maybe time or state-invariant?

Reply:
Ah yes maybe `Constant (time-invariant) parameters`.